### PR TITLE
[Data First] Styling search box

### PR DESCRIPTION
**Pull request by yebrahim**
image

### CODE REVIEW

File: sources/web/datalab/polymer/components/datalab-data/datalab-data.ts [hunk 1]
```diff
@@ -65,12 +67,25 @@ class DataElement extends Polymer.Element {
...
+      clearInterval(this._searchInputDelayTimeout);
+      this._searchInputDelayTimeout = 0;
+    }
+
+    this._searchInputDelayTimeout = setTimeout(() => {
+      // TODO - clearing the resultsList may cause unnecessary refreshes, clean
+      // this up when we figure out how we actually want to handle the search
+      // call.
+      this._resultsList = [];
+      this._sendQuery(this._searchValue, this._handleQueryResults.bind(this));
```
Comment (jimmc) [inline sources/web/datalab/polymer/components/datalab-data/datalab-data.ts]: I think you should clear this._searchInputDelayTimeout in this block.

**Comment by yebrahim** [comment]
I actually thought that was the intended behavior of the original code, I did not notice the search icon was actually a button. :)

Let me do that instead and make the button more clear.

**Comment by yebrahim** [comment]
Reusing this PR to revert to the search button and make it more obvious. PTAL.

**Comment by yebrahim** [comment]
PTAL now.

**Comment by yebrahim** [comment]
How about now? :)

**Pull request merged**